spec/helpers/overlay_helpers.rb

```diff
@@ -146,5 +146,10 @@ def nth_panel(root, n)
     def nth_ccol(root, n)
       root.find(".ccol:nth-child(#{n})")
     end
+
+    def manual_close_facet_list
+      page.find('#master-overlay-parent a.master-overlay-close').click
+      sleep 0.5 # wait for animation
```

Comment:
I don't like sleeps in our tests. Use synchronize instead so you wait exactly as long as needed. Also, I'm not sure there are animation in test mode (I can't remember if they're shut off for CSS or just in JS)